tox.ini

```diff
@@ -22,6 +22,7 @@ deps = pre-commit
 commands = pre-commit run --all-files --show-diff-on-failure
 
 [flake8]
+ignore = E203
```

Comment:
I see not reason for this, also no explanation. If is a bug in flake8, why not using an inline exclude instead of diabling an entire rule?

Reply:
Afaik, the rule is broken full stop, not in particular cases. It's also not PEP8 compliant.